modules/gha/examples/go/main.go

```diff
@@ -44,7 +44,7 @@ func (m *Examples) GhaGithubContext() *dagger.Directory {
 			dag.Gha().Workflow("lint all branches", dagger.GhaWorkflowOpts{
 				OnPush: true,
 			}).
-				WithJob(dag.Gha().Job("lin", "lint --source=${GITHUB_REPOSITORY_URL}#${GITHUB_REF}")),
+				WithJob(dag.Gha().Job("lin", "lint --source=${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}#${GITHUB_REF}")),
```

Comment:
`lin` is a typo (same as before this change), not blocking, something to come back.

Reply:
Oops, missed this somehow - fixed it while in the area.